java/memory/src/main/java/org/apache/arrow/memory/ArrowByteBufAllocator.java

```diff
@@ -146,4 +147,45 @@ private RuntimeException fail() {
     throw new UnsupportedOperationException("Allocator doesn't support heap-based memory.");
   }
 
+  /**
+   * This method was copied from AbstractByteBufAllocator. Netty 4.1.x moved this method from
```

Comment:
Does it make sense to change:
```
public class ArrowByteBufAllocator implements ByteBufAllocator
```
to
```
public class ArrowByteBufAllocator extends AbstractByteBufAllocator
```
?

Reply:
@icexelloss yeah, good point. I made the change